interface/patient_tracker/patient_tracker_status.php

```diff
@@ -1,126 +1,129 @@
 <?php
-/** 
- * Patient Tracker Status Editor 
+/**
+ * Patient Tracker Status Editor
  *
  * This allows entry and editing of current status for the patient from within patient tracker and updates the status on the calendar.
  * Contains a drop down for the Room information driven by the list Patient Flow Board Rooms.
- * 
- * Copyright (C) 2015-2017 Terry Hill <teryhill@librehealth.io> 
- * 
+ *
+ * Copyright (C) 2015-2017 Terry Hill <teryhill@librehealth.io>
+ *
  * LICENSE: This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0
- * See the Mozilla Public License for more details. 
+ * See the Mozilla Public License for more details.
  * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
- * 
- * @package LibreEHR 
- * @author Terry Hill <teryhill@librehealth.io> 
- * @link http://www.libreehr.org 
- *  
+ *
+ * @package LibreEHR
+ * @author Terry Hill <teryhill@librehealth.io>
+ * @link http://www.libreehr.org
+ *
  * Please help the overall project by sending changes you make to the author and to the LibreEHR community.
- * 
- */ 
- 
-$fake_register_globals=false;
-$sanitize_all_escapes=true;
-  
+ *
+ */
+
+$fake_register_globals = false;
+$sanitize_all_escapes = true;
+
 require_once("../globals.php");
 require_once("$srcdir/options.inc.php");
 require_once("$srcdir/forms.inc");
 require_once("$srcdir/encounter_events.inc.php");
 require_once("$srcdir/patient_tracker.inc.php");
 ?>
- <html>
-  <head>
-  <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-  <link rel="stylesheet" type="text/css" href="../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
-  <script type="text/javascript" src="../../library/js/jquery.1.3.2.js"></script>
-  <script type="text/javascript" src="../../library/js/common.js"></script>
-  <script type="text/javascript" src="../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
+<html>
+<head>
+    <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+    <link rel="stylesheet" type="text/css" href="../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen"/>
+    <script type="text/javascript" src="../../library/js/jquery.1.3.2.js"></script>
```

Comment:
I would go ahead and change this to 1.9.1 and change it in the main tracker file